eval.py

```diff
@@ -2,61 +2,96 @@
 
 import hydra
 import torch
+from omegaconf import OmegaConf
 
 from train import val_epoch
 from model import make_model
+from utils import translate_sample_sentences
 from modules.label_smoothing import LabelSmoothing
 from data.dataloaders import load_vocab, load_tokenizers, create_dataloader
 
 
 @hydra.main(version_base=None, config_path="config", config_name="eval_conf")
 def main(cfg):
-    checkpoint = load_checkpoint(run_id=cfg.run_id, epoch=cfg.model_epoch, device=cfg.device)
-    model_cfg = checkpoint["cfg"]
+    checkpoint = load_checkpoint(
+        run_name=cfg.run_name,
+        epoch=cfg.model_epoch,
+        device=cfg.device,
+    )
+    train_cfg = OmegaConf.create(checkpoint["cfg"])
     spacy_de, spacy_en = load_tokenizers()
-    vocab_src, vocab_tgt = load_vocab(spacy_de, spacy_en, model_cfg["vocab_data_slice"])
+    vocab_src, vocab_tgt = load_vocab(spacy_de, spacy_en, train_cfg)
```

Comment:
Wouldn't it be better to only pass max_tokens parameter to this dunction? It is the only hyperparameter it uses.